### DISCUSSION TITLE
v0.10.9

### DISCUSSION THREAD

**Discussion by AsamK** [AsamK/signal-cli#982] [Releases]
## New Contributors
* @JohnRush made their first contribution in https://github.com/AsamK/signal-cli/pull/972

**Full Changelog**: https://github.com/AsamK/signal-cli/compare/v0.10.8...v0.10.9

<hr /><em>This discussion was created from the release <a href='https://github.com/AsamK/signal-cli/releases/tag/v0.10.9'>v0.10.9</a>.</em>

**Comment by LFrank2021** [comment]
Is it okay to ask questions here?
I tried to compile signal-cli 0.10.9 and libsignal-client.jni v0.19.0 (and v0.19.1) myself and get the following error:
```Error loading state file for user <NUMBER>: org.signal.libsignal.zkgroup.InvalidInputException: Deserialization failure in zkgroup (AssertionError)```

An older issue points at an incompatibility from signal-cli and libsignal-client.jni.
If so, which libsignal-client is needed for v0.10.9?

Or can this be related to libsignal-client.jni being generated by using ```scl enable devtoolset-9 bash``` to get a newer gcc and using gcc 4.8.1 for signal-cli?

**Reply by AsamK** [reply]
Sure, signal-cli 0.10.9 requires libsignal-client 0.18.1
Also see the documentation in the wiki: https://github.com/AsamK/signal-cli/wiki/Provide-native-lib-for-libsignal#determine-the-required-libsignal-client-version

**Reply by LFrank2021** [reply]
Thank you.
Since I am having another assertion, I will open an issue. I think it is better to analyse there.
